bot/check-test.sh

```diff
@@ -13,8 +13,203 @@ job_dir=${PWD}
 job_out="slurm-${SLURM_JOB_ID}.out"
 job_test_result_file="_bot_job${SLURM_JOB_ID}.test"
 
+# Check that job output file is found
+[[ ${VERBOSE} -ne 0 ]] && echo ">> searching for job output file(s) matching '"${job_out}"'"
+if  [[ -f ${job_out} ]]; then
+    SLURM=1
```

Comment:
We can, and its a good idea, but I think we should also change it in `check-build.sh` then :) I'll do that, can include it in this PR since its a straightforward change.

Reply:
Done